lib/twitter-ads/request.rb

```diff
@@ -57,12 +60,22 @@ def perform
 
     def oauth_request
       request  = http_request
-      domain   = @options.fetch(:domain, DEFAULT_DOMAIN)
+      default_domain = @sandbox_mode ? SANDBOX_DOMAIN : DEFAULT_DOMAIN
```

Comment:
I'll bundle up this logic into a `domain` method. In the next release, we'll also begin supporting TON API uploads to ton.twitter.com and media uploads to upload.twitter.com and will need a more robust solution here for determining which domain to use.